### ISSUE TITLE
Add Multiroom Join as an option for Power on via setting

### ISSUE THREAD

**Issue by mbolo01** [connectjunkie/homebridge-beoplay#10]
Multiroom join is a B&O feature that enables a product to play in sync what another B&O product is playing.

When multiroom is configured in a B&O product, powering on a product triggers the join attempt first, and if no other product is playing, defaults to the last played source.

Having this option as a parameter to "power on via setting" in the plug-inwould be very useful.

**Comment by connectjunkie** [comment]
Initial support in 1.0.1. This seems to work as far as I can test with my setup - can you verify?

**Comment by mbolo01** [comment]
Works ok here too with two products, well done!! and thanks. I'll add more products and continue testing.

Not sure if this is reproducible with any object type (bulb, fan, TV), but here is another behavior related to the "join" command that could be interesting to add: 

- Assume you have 3 B&O products, A, B & C. 
- A is playing RadioA, B is playing RadioB. 
- When starting product C to join a playing product, it will join A or B depending on which one advertised itself the latest (can't give you more detailed information, I don't have them :-)). 
- Assuming C has joined B when starting, and that you want to join A. With current B&O products, or B&O minimalist physical remotes, you can press twice the start button to cycle between the product sources, including other playing products to start with.

With app button limitations, the only way I can see it (from a non developper angle) is to leverage the long press to expose another action alongside the "volume" bar, like for bulbs that can have multiple colors choices, so that pressing this extra button would cycle between sources (the ones defined in the product itself or other products playing).

Note: It is a pity that speaker type cannot be used with speakers and that TV type has Siri power on/off limitations. This really reduces the opportunities to enhance your integration. For now I'm using Bulb type for the B&O speakers.

**Comment by connectjunkie** [comment]
Let me look into this, however off the top of my head I can't think of an elegant way of implementing that in a HomeKit context. For additional functionality I'm already looking at what I need to add additional switches and toggles to in order to support additional functionality, but I've resisted that so far as it can get very ugly very quickly if not planned properly. 

One thing to note - I have been looking at implementing the HomeKit Smart Speaker as a device type option, which is still pretty limited (although not as limited as the normal speaker), but would at least give you the ability to see the currently playing track and some play/pause controls in a native HomeKit context. I've just opened a new issue to track that as a planned enhancement.
